bioc/eset_sumexp.Rmd

```diff
@@ -14,29 +14,33 @@ library(GEOquery)
 ```
 
 ```{r}
-geoq <- getGEO("GSE9514")
+geoq <- getGEO("GSE9514")    # download a dataset from GEO
 names(geoq)
-e <- geoq[[1]]
+e <- geoq[[1]]    # extract ExpressionSet
 ```
 
 ### ExpressionSet
 
 ```{r}
-dim(e)
+# exprs gives expression matrix
+dim(e)    # number of features and samples in ExpressionSet
 exprs(e)[1:3,1:3]
-dim(exprs(e))
+dim(exprs(e))    # rows are features, columns are samples
 
+# pData gives phenotype data (sample information)
 pData(e)[1:3,1:6]
```

Comment:
same idea here -- subset the object before extracting ... a good habit when the data are in an external source